### PR TITLE
Improve OCW Next build script

### PR DESCRIPTION
**Pull request by markbreedlove**
* Do our own `aws s3 sync` of the source data bucket, not using `ocw-to-hugo`
* Clear `node_modules` each time for extra peace of mind regarding dependencies
* Clear out `site/content` directory in case of bad old-version markdown

### CODE REVIEW

**Comment by blarghmatey** [comment]
:+1:

**Comment by pdpinch** [comment]
@gumaerc can you take a look at this too?

File: salt/apps/ocw/templates/webhook-publish.sh.jinja [hunk 1]
```diff
@@ -55,10 +59,23 @@ git pull || error_and_exit "Can not pull hugo-course-publisher"
...
     || error_and_exit "Can not install hugo-course-publisher"
 
-log_message "Pulling ocw-to-hugo and source data"
+log_message "Clearing site/content directory"
+rm -rf site/content/*
+if [ $? -ne 0 ]; then
+    error_and_exit "Could not clear site/content directory"
+fi
+
+log_message "Clearing node_modules directory"
```
Comment (gumaerc) [inline salt/apps/ocw/templates/webhook-publish.sh.jinja:68]: This is great, but I think we need to run `yarn install` after clearing out `node_modules` to ensure that everything is installed before the import / build.

**Review by gumaerc — CHANGES_REQUESTED** [review]
Just one thing:

File: salt/apps/ocw/templates/webhook-publish.sh.jinja (hunk 1 above)
Comment (markbreedlove) [reply salt/apps/ocw/templates/webhook-publish.sh.jinja:68]: OK. I got those things out of order. Will re-order them.

Comment (markbreedlove) [reply salt/apps/ocw/templates/webhook-publish.sh.jinja]: Changed in commit 00bfde3d

**Review by gumaerc — APPROVED** [review]
:+1:

**Pull request merged**